### PR TITLE
Bleeding edge

### PR DESCRIPTION
**Pull request by sperky7933**
adds goontext
removes the third gender from preferences screen (plural preference)
removes station engineers having industrial welders (????)
made the settings.txt sane, made the admins.txt accurate

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
🤨

File: config/admins.txt [hunk 1]
```diff
@@ -4,134 +4,4 @@
...
-octareenroon91 = Game Master
-Serpentarium = Game Master
-Averagejoe82 = Game Master
-The Dreamweaver = Game Master
-Denton-30 = Game Master
-Naksuasdf = Game Master
-MrDoomBringer = Game Master
-shizcalev = Game Master
-NicBR = Game Master
+Sperky7933 = Host
```
Comment (JohnFulpWillard) [inline config/admins.txt:7]: 🤨

Comment (not-a-furry) [reply config/admins.txt:7]: u have to ask pretty please 4 admin

**Comment by not-a-furry** [comment]
oh wait i pushed this to the wrong brancH HAHAHAHA

**Comment by Imaginos16** [comment]
🤨

**Comment by not-a-furry** [comment]
yeah sorry iddnt mean to merge the branch upstream, im new to using github desktop UI

**Comment by DrDiasyl** [comment]
What is this PR?

**Comment by ArcaneMusic** [comment]
image
Pim, can I, ugh, can I talk to you for a second? Look, Pim, I know it's our job to help this guy's code and everything, but I think this guy is a lost cause. He has obviously made up his mind. Why don't we just cut our losses and close this PR?

File: code/__HELPERS/unsorted.dm [hunk 2]
```diff
@@ -453,11 +453,12 @@ Turf and target are separate in case you want to teleport some distance from a t
 	var/x = min(world.maxx, max(1, A.x + dx))
 	var/y = min(world.maxy, max(1, A.y + dy))
 	return locate(x,y,A.z)
-
+/*
 /proc/arctan(x)
 	var/y=arcsin(x/sqrt(1+x*x))
 	return y
-
+Will not compile with this uncommented.
```
Comment (Iamgoofball) [inline code/__HELPERS/unsorted.dm:460]: have you considered fixing your code?

**Comment by EdgeLordExe** [comment]
what

**Comment by ATH1909** [comment]
> removes the third gender from preferences screen (plural preference)

cringe

**Comment by JohnFulpWillard** [comment]
idk why this pr is still up if you said it wasnt even for this repo

**Comment by Theophrastus1** [comment]
sasuga /vm/

**Comment by LemonInTheDark** [comment]
Fucking 4chan downstreams fucking downstreams fucking hhhhhhhhhhhh

**Comment by Mothblocks** [comment]
Yoooo html prefs are back

**Pull request closed without merging**